cpp/ql/src/Security/CWE/CWE-326/InsufficientKeySize.ql

```diff
@@ -0,0 +1,56 @@
+/**
+ * @name Use of a cryptographic algorithm with insufficient key size
+ * @description Using cryptographic algorithms with too small a key size can
+ *              allow an attacker to compromise security.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id cpp/insufficient-key-size
+ * @tags security
+ *       external/cwe/cwe-326
+ */
+
+import cpp
+import semmle.code.cpp.ir.dataflow.DataFlow
+import semmle.code.cpp.ir.IR
```

Comment:
Given the number of re-evaluation issues we've had in the past, I wouldn't be surprised if we need to import GVN before importing the IR here. But let's revisit that once we have a DCA run.

Reply:
Thank you so much

Reply:
DCA doesn't look like there's reevaluation. I see a 2-4% slowdown depending on project. 4% is pretty high, I'll see if I can make some small changes to improve it.